docs/sources/explore/correlations-editor-in-explore.md

```diff
@@ -0,0 +1,118 @@
+---
+labels:
+  products:
+    - enterprise
+    - oss
+title: Correlations Editor in Explore
+weight: 400
+---
+
+# Correlations Editor in Explore
+
+{{% admonition type="note" %}}
+The Explore editor is available in 10.1 and later versions. In the editor, transformations is available in Grafana 10.3 and later versions.
+{{% /admonition %}}
+
+Correlations allow users to build a link between any two datasources. For more information about correlations in general, please see the [documentation]({{< relref "../administration/correlations" >}}) about building correlations in the administration page.
+
+## Create a correlation
+
+1. Go to Explore page.
+1. Select a data source that you would like to be [the source data source]({{< relref "../administration/correlations/correlation-configuration#source-data-source-and-result-field" >}}) for a new correlation.
+1. Run a query producing data in [a supported visualization]({{< relref "../administration/correlations#correlations" >}}).
+1. Click "+ Add" button in the top toolbar and select "Add correlation" (you can also select "Correlations Editor" from the [Command Palette]({{< relref "../search#command-palette" >}})).
+1. Explore is now in Correlations Editor mode indicated by a blue border and top bar. You can exit Correlations Editor using the Exit button in the top bar.
+1. The visualization is enriched with links allowing to create new correlations. Links are attached to the data that you can use to build a new query:
+   - Logs: links are displayed next to field values inside log details for each log row
+   - Table: every table cell is a link
+1. Click on a link to begin adding new correlation. Links are associated with a field that will be used as [a results field of a correlation]({{< relref "../administration/correlations/correlation-configuration" >}}). This means the correlation link will be created on the link you select once the correlation is saved.
+1. Explore opens a split view. Use the right pane to setup [the target query source of the correlation]({{< relref "../administration/correlations/correlation-configuration#target-query" >}}).
+1. Build target query using [variables syntax]({{< relref "../dashboards/variables/variable-syntax" >}}) with variables from the list provided at the top of the pane. The list contains sample values from the selected data row.
+1. Provide optional label and description. Label is used as the name of the link inside the visualization. It can contain variables.
+1. Click "Save" button in the top bar to save the correlation and exit Correlations Editor mode.
+1. The link used to create the correlation will be replaced with a data link in each row. The variables will be dynamically replaced with values from a selected row.
```

Comment:
Avoid "will" unless talking about future behavior: https://developers.google.com/style/word-list#will

I don't think I understand the second sentence: The variables are dynamically replaced with values from a selected row.

```suggestion
1. The link used to create the correlation is replaced with a data link in each row. The variables are dynamically replaced with values from a selected row.
```

Reply:
> I don't think I understand the second sentence: The variables are dynamically replaced with values from a selected row.

This is the most complicated idea about correlations to get across and I'm struggling how to construct a concise sentence describing it without an example.

Basically you end up with data with a button alongside it. 
Like `field1=hello field2=world`

Building the correlation, you construct a query with a variable
`SELECT * FROM ${field1} WHERE name=${field2}`

When you click the button, data next to the button goes into the constructed query. 
`SELECT * FROM hello WHERE name=world`

Maybe something like

```The link used to create the correlation is replaced with a data link in each row. When the link is clicked, the query defined will run in another pane, with the variables replaced dynamically with the values from the selected row.```

Reply:
I like that explanation a lot. I can see that it's a complicated thing to explain written down but at least from my perspective as an engineer, that rewrite makes sense to me!